<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MoreUiClass</class>
 <widget class="QDialog" name="MoreUiClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>更多信息</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../.res/QuickInput.qrc">
    <normaloff>:/icon.png</normaloff>:/icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">*{color:black;border:none;outline:none;font-size:12px;font-family:&quot;Microsoft YaHei&quot;}QMainWindow,QDialog{border:1px solid gray}#titleWidget{background-color:white}#titleWidget *{font-size:14px}#titleWidget QPushButton,#titleWidget QCheckBox::indicator{background-color:#CCC;width:20px;height:20px;border-radius:10px}#titleWidget QCheckBox::indicator:checked{image:url(:/checked.png)}#clientWidget{background-color:#CEF}QMenu,#clientWidget QLineEdit,#clientWidget QTextEdit,#clientWidget QKeyEdit,#clientWidget QCheckBox::indicator,#clientWidget QRadioButton::indicator,#clientWidget QHeaderView,#clientWidget QHeaderView::section,#clientWidget QTableWidget,#clientWidget QTableWidget QTableCornerButton::section,#clientWidget QComboBox,#clientWidget QComboBox QAbstractItemView{background-color:white;color:black}#clientWidget QPushButton,#clientWidget QTabBar::tab,#clientWidget QTabBar::tab:first,#clientWidget QTabBar::tab:last{background-color:#ADE;color:black}#clientWidget QCheckBox::indicator,#clientWidget QRadioButton::indicator{width:20px;height:20px}#clientWidget QComboBox{padding-left:3px}QMenu::item:selected,#clientWidget QPushButton:hover,#clientWidget QCheckBox::indicator:hover,#clientWidget QRadioButton::indicator:hover,#clientWidget QTableWidget::item:hover,#clientWidget QTabBar::tab:hover,#clientWidget QTabBar::tab:first:hover,#clientWidget QTabBar::tab:last:hover{background-color:#BEF;color:black}#clientWidget QTableWidget::item:selected{background-color:#CEF;color:black}#clientWidget QTabBar::tab:selected,#clientWidget QTabBar::tab:first:selected,#clientWidget QTabBar::tab:last:selected{background-color:#CEF;color:black}#clientWidget QCheckBox::indicator:checked,#clientWidget QRadioButton::indicator:checked{image:url(:/checked.png)}#clientWidget QPushButton:disabled,#clientWidget QCheckBox:disabled,#clientWidget QRadioButton:disabled{color:gray}#clientWidget #bnWndActive,#clientWidget #bnPos,#clientWidget #bnColorRect,#clientWidget #bnImageRect,#clientWidget #bnImageShot{background-color:white}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <widget class="QWidget" name="titleWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QLabel" name="lbIcon">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background:none;border-image:url(:/icon.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbTitle">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>20</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Microsoft YaHei</family>
          <pointsize>-1</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background:none;font-family:&quot;Microsoft YaHei&quot;;
font-size:14px;</string>
        </property>
        <property name="text">
         <string>更多</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="titleSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="bnClose">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>30</horstretch>
          <verstretch>20</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="toolTip">
         <string>关闭</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{background-color:#E00;width:20px;height:20px;border-radius:10px;}
QPushButton:hover{background-color:#C00;}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="clientWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QLabel" name="lbHowUse">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Microsoft YaHei</family>
          <pointsize>-1</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">padding-bottom:5px;font-size:14px;</string>
        </property>
        <property name="text">
         <string>使用方法</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QScrollArea" name="saHowUse">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QScrollArea{background:transparent;border:1px solid gray;padding:10px}</string>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaTip">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>756</width>
           <height>1530</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QWidget{background:transparent}</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="lbHowUseText">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>Microsoft YaHei</family>
              <pointsize>-1</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>➤CPS计算：1000 ÷ 间隔时间，50CPS = 1000 ÷ 20


————————————————————基本设置————————————————————

➤初次打开时：切换 [设置] ➜ [开启/停止键] ➜ 点击输入框后按下所需按键

➤简单设置连点器：切换 [功能] ➜ 勾选快捷连点 ➜ 点击输入框并按下所需按键 ➜ 输入间隔时间（毫秒）越小越快 ➜ 选择模式

➤通过宏实现更多功能：
ㅤㅤ1.切换 [宏] ➜ 使用录制 / 添加：
ㅤㅤㅤㅤ[录制]：点击后将记录按键与鼠标位置 ➜ 完成后关闭将自动保存一个宏
ㅤㅤㅤㅤ[添加]：点击后将添加一个空宏 ➜ 选中宏后点击编辑 ➜ 添加所需的功能 ➜ 完成后关闭窗口将自动保存宏
ㅤㅤ2.切换 [触发]：
ㅤㅤㅤㅤ选中一个宏 ➜ 点击禁用变为启用 ➜ 设置模式 ➜ 点击输入框后按下所需按键 ➜ 输入次数

➤开始使用：按下 [开启/停止键] 并且弹出 [已启用] ➜ 按下快捷连点 / 宏的设置的快捷键


————————————————————编辑窗口————————————————————

➤键盘/鼠标：
ㅤㅤ1.点击输入框并按下一个键盘或鼠标按键
ㅤㅤ2.选择需要的操作，按下，松开，点击

➤鼠标移动：
ㅤㅤ1.在X，Y输入坐标或距离，X是水平方向，Y是垂直方向
ㅤㅤ1.1在随机输入的范围按照X，Y计算，比如10，10，2：结果是8，8 ~ 12，12（如果需要精准点击则不需要输入）
ㅤㅤ2.勾选位置，鼠标指针将会移动到对应的坐标；X，Y的最大值是10000，点击右侧的[+]可以获取坐标
ㅤㅤ3.勾选移动，鼠标指针将会根据上一次的位置移动一段距离；X，Y按照屏幕分辨率来设置移动距离

➤等待：
ㅤㅤ1.在时长输入所需时长，单位毫秒
ㅤㅤ2.根据所需输入随机，随机按照时长计算，比如10，2：结果是8 ~ 12

➤循环：
ㅤㅤ1.输入循环次数
ㅤㅤ2.添加后再点击一下宏项目，点击编辑，方法和外面一致

➤复制文字
ㅤㅤ1.在编辑框输入所需的文本
ㅤㅤ2.执行后文本将复制到剪贴板，通过Ctrl + V的方式粘贴到对应的位置

➤查找颜色：
ㅤㅤ1.点击范围最右侧的[+]，框选需要查找的范围
ㅤㅤ2.点击颜色最右侧的[  ]，选择需要的颜色
ㅤㅤ3.若识别不到则添加误差，误差越大越容易识别成功
ㅤㅤ4.添加后再点击一下宏项目，点击编辑，方法和外面一致

➤查找图像：
ㅤㅤ1.点击范围最右侧的[+]，框选需要查找的范围
ㅤㅤ2.点击截图[+]，截取目标图像，建议左上角位置在目标内
ㅤㅤ3.若识别不到则减少相似，相似越小越容易识别成功
ㅤㅤ4.添加后再点击一下宏项目，点击编辑，方法和外面一致


—————————————————编辑窗口(窗口模式)——————————————————

➤窗口模式可以在后台时控制窗口，但兼容性较差，如果程序不完全通过窗口消息来控制 或 程序需要验证是否处于前台
ㅤ比如部分游戏可以在游戏中用窗口模式，但切换到其他程序没有效果

➤使用方法
ㅤㅤ1.勾选标题栏左侧的 [窗口模式（后台）] 之后点击右侧的+选择窗口
ㅤㅤ2.鼠标位置，颜色范围会根据窗口来定位
ㅤㅤ3.在执行鼠标点击之前需要添加鼠标位置


————————————————————触发模式————————————————————

➤切换：点击触发键执行宏，再次点击触发键停止

➤按下：按下触发键执行宏，直到完成对应次数或松开

➤松开：松开触发键执行宏，直到完成对应次数或按下


————————————————————其他功能————————————————————

➤窗口控制状态：点击 [+] 并根据提示操作，之后当切换到对应窗口才会启用连点（先用开关键启用连点器）

➤显示当前时间：输入一个按键，之后点击会弹出当前时间


————————————————————常见问题————————————————————

➤不能一直循环，或只能运行一次：
ㅤㅤ在宏编辑界面删除 [循环]（此循环不能让整个宏循环），然后在触发里面的次数输入0（此处控制整个宏的次数）

➤设置了按键连点，但结束连点时按键可能是按下的：
ㅤㅤ在宏编辑界面选择 [结束时] ，添加对应按键的松开</string>
            </property>
            <property name="textFormat">
             <enum>Qt::PlainText</enum>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbLog">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Microsoft YaHei</family>
          <pointsize>-1</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">padding-top:10px;padding-bottom:5px;font-size:14px;</string>
        </property>
        <property name="text">
         <string>更新记录</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QScrollArea" name="saLog">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>300</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QScrollArea{background:transparent;border:1px solid gray;padding:10px}</string>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaLog">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>756</width>
           <height>2611</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QWidget{background:transparent}</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>10</number>
          </property>
          <item>
           <widget class="QLabel" name="lbLogText">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>Microsoft YaHei</family>
              <pointsize>-1</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>2024-5-8
ㅤㅤ1：增加查找图像
ㅤㅤ2：增加弹出文字
ㅤㅤ3：增加编辑的测试运行
ㅤㅤ4：增加宏列表的重新加载，取消每次编辑的无效加载
2024-5-4
ㅤㅤ1：增加恢复鼠标位置
2024-4-29
ㅤㅤ1：增加主题
2024-4-11
ㅤㅤ1：增加编辑列表的右键菜单
2024-4-7
ㅤㅤ1：UI（选框不清楚是否勾选）
ㅤㅤ2：优化性能
2024-3-23
ㅤㅤ1：增加多选，可用于删除和复制
ㅤㅤ2：增加在临时文件夹运行时自动提示安装
2024-3-14
ㅤㅤ1：修复宏列表为空时无法正常录制
ㅤㅤ2：优化窗口模式，可以通过位置自动查找子窗口
2024-3-7
ㅤㅤ1：增加随机循环
2024-3-6
ㅤㅤ1：增加窗口录制时检测目标窗口是否有效
ㅤㅤ2：增加选择窗口时按ESC取消
2024-2-28
ㅤㅤ1：优化触发判断速度
ㅤㅤ2：修改时保留备注
2024-2-22
ㅤㅤ1：增加结束时执行的宏，可用于在宏结束时松开按键，确保每次结束时按键都是松开的
2024-2-8
ㅤㅤ1：增加按键状态判断，可实现用一个按键控制一个按键的长按切换或组合键判断
2024-1-30
ㅤㅤ1：编辑宏使用Ctrl C，Ctrl V可复制粘贴
2024-1-8
ㅤㅤ1：快捷连点附带随机
2024-1-7
ㅤㅤ1：增加窗口输入（后台）模式
2024-1-3
ㅤㅤ1：优化屏蔽触发键，能屏蔽键盘自动重复输入
ㅤㅤ2：增加任务栏右键菜单
2023-12-31
ㅤㅤ1：优化查找颜色速度
2023-12-30
ㅤㅤ1：增加默认启用
ㅤㅤ2：增加默认参数
2023-12-25
ㅤㅤ1：增加鼠标位置和查找颜色的位置预览
ㅤㅤ2：优化内存管理
2023-12-24
ㅤㅤ1：修改json动态分配结构，不兼容以前版本
ㅤㅤ2：增加查找颜色的找到并移动
ㅤㅤ3：宏触发直接点击列表启用或禁用
2023-12-21
ㅤㅤ1：修复创建开机任务（在使用电池时不启动）
ㅤㅤ2：修复移动无法输入负数错误
2023-12-17
ㅤㅤ1：增加导入导出
ㅤㅤ2：增加松开触发
2023-12-14
ㅤㅤ1：增加屏蔽触发键（仅窗口应用有效）
ㅤㅤ2：修复时间过大导致内存溢出
2023-12-11
ㅤㅤ1：优化界面
ㅤㅤ2：修复编辑界面删除宏导致崩溃
2023-12-9
ㅤㅤ1：优化界面
2023-12-8
ㅤㅤ1：优化列表间距
2023-12-5
ㅤㅤ1：增加移动宏功能
ㅤㅤ2：优化宏列表备注
2023-12-2
ㅤㅤ1：增加备注功能
2023-11-30
ㅤㅤ1：修复Ctrl,Alt+数字组合键
ㅤㅤ2：调整界面
2023-11-29
ㅤㅤ1：增加结束和结束循环功能
ㅤㅤ2：修复颜色框选错误
2023-11-25
ㅤㅤ1：增加无限嵌套功能
ㅤㅤ2：增加结束功能
2023-11-24
ㅤㅤ1：增加查找颜色未找到的处理
2023-11-21
ㅤㅤ1：优化列表效率
2023-11-16
ㅤㅤ1：增加随机移动
2023-11-4
ㅤㅤ1：优化列表，UI缩放
2023-11-3
ㅤㅤ1：增加窗口控制状态，详细看使用方法
2023-11-2
ㅤㅤ1：增加自定义窗口放大
2023-10-30
ㅤㅤ1：优化性能，0ms以上的CPU占用更低
2023-10-29
ㅤㅤ1：增加触发的次数，可指定宏的执行次数
ㅤㅤ2：增加编辑的循环，可指定宏的执行次数
2023-10-28
ㅤㅤ1：修复CPS不准确，提高间隔时间的精度，但在16ms以下会增加CPU占用
2023-10-27
ㅤㅤ1：过滤录制的键盘重复按下
ㅤㅤ2：修复单个触发键会被其他键中断
ㅤㅤ3：增加编辑窗口宏列表使用Shift ↑↓移动宏位置，Shift ←删除宏
2023-10-21
ㅤㅤ1：增加录制的开始按钮
2023-10-20
ㅤㅤ1：修复组合触发键的误触发
2023-10-17
ㅤㅤ1：增加启用/停止组合键
ㅤㅤ2：修复颜色识别卡顿
2023-10-16
ㅤㅤ1：增加快捷连点模式
2023-10-14
ㅤㅤ1：增加显示时间开关
2023-10-10
ㅤㅤ1：修复缩放识别位置错误
2023-10-9
ㅤㅤ1：增加组合触发键，可输入两个按键来触发，也可输入一个
2023-10-5
ㅤㅤ1：修复缩放字体显示过大
2023-10-5
ㅤㅤ1：增加范围颜色查找功能
ㅤㅤ2：优化获取颜色，获取坐标
2023-10-5
ㅤㅤ1：修复查找颜色错误
2023-10-4
ㅤㅤ1：增加查找颜色功能，详细看使用方法
ㅤㅤ2：优化列表显示效果
2023-10-3
ㅤㅤ1：增加编辑窗口的修改功能，双击列表的项目即可修改
ㅤㅤ2：增加多开功能，复制程序的文件夹就可以运行多个
2023-10-2
ㅤㅤ1：优化弹窗刷新，从透明度更改时刷新改为仅内容更改时刷新，提高效率
2023-9-30
ㅤㅤ1：使用Qt5.15.2并重新编译为32位程序、
ㅤㅤ2：增加宏触发状态显示，在屏幕的顶部将弹出提示
2023-9-28
ㅤㅤ1：增加显示当前时间功能
2023-9-27
ㅤㅤ1：增加快捷连点的间隔时间功能
ㅤㅤ2：修复快捷点击按键与宏触发键冲突导致程序失效
2023-9-24
ㅤㅤ1：增加随机等待功能，模拟真实点击时的速度差异
ㅤㅤ2：增加获取位置功能，点击获取后再点击目标位置
2023-9-23
ㅤㅤ1：修复按下启用/停止键时按下其他键异常
ㅤㅤ2：增加对滚轮的支持
ㅤㅤ3：发布到bilibili
2023-9-21
ㅤㅤ1：第一个版本</string>
            </property>
            <property name="textFormat">
             <enum>Qt::PlainText</enum>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../.res/QuickInput.qrc"/>
 </resources>
 <connections/>
</ui>
